packages/web-components/fast-components/docs/color.md

```diff
@@ -0,0 +1,143 @@
+# Color
+FAST utilizes an adaptive color system that provides some unique advantages:
+1. Ensure text meets [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) contrast requirements.
+2. Easily swap from light mode to dark, or anywhere in-between.
+3. Color themeing through palette tinting.
+4. Perceptually uniform UI across different surface colors.
+
+## Algorithmic colors
+To understand the FAST color system, the first thing to understand is that all of the named colors enumerated below are algorithmic; they are a product of the *designSystem* object in which they are calculated.
+
+### Inputs
+Each color recipe expects as its sole argument the FAST *DesignSystem* object, but there are a few core pieces of data from that object that impact color resolution.
+
+### Palettes
+Each color recipe operates on a palette. A palette in an array of hexadecimal colors ordered from light to dark. By default, FAST components leverage the `neutralPalette` and the `accentPalette`. 
+
+### Background color
+This is the contextual color in which the recipe is rendering on. In foreground, outline, and divider recipes it is used the target color in which to ensure contrast against. In fill recipes it is sometimes used as the starting location in the appropriate palette to begin resolution.
```

Comment:
nit: can you update the first two sentences to be "This is the contextual color that the recipe uses to determine what color it is rendering on. In foreground, outline, and divider recipes it will use this color to ensure that it creates an accessible color that meets contrast requirements."